src/main/resources/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm/config.jelly

```diff
@@ -66,14 +65,10 @@ THE SOFTWARE.
       <f:textbox name="ldap.groupNameAttribute" value="${instance.groupNameAttribute}" />
     </f:entry>
     <f:entry title="${%Manager DN}" >
-      <f:textbox name="managerDN" value="${instance.managerDN}" autocomplete="off"
-         checkUrl="'${rootURL}/securityRealms/LDAPSecurityRealm/serverCheck?field=managerDN&amp;server='+encodeURIComponent(this.form.elements['ldap.server'].value)+'&amp;managerDN='+encodeURIComponent(this.value)+'&amp;managerPassword='+encodeURIComponent(this.form.elements['ldap.managerPassword'].value)"
-      />
-    </f:entry>
-    <f:entry title="${%Manager Password}" >
-      <f:password name="managerPassword" value="${instance.managerPassword}" autocomplete="off"
-         checkUrl="'${rootURL}/securityRealms/LDAPSecurityRealm/serverCheck?field=password&amp;server='+encodeURIComponent(this.form.elements['ldap.server'].value)+'&amp;managerDN='+encodeURIComponent(this.form.elements['ldap.managerDN'].value)+'&amp;managerPassword='+encodeURIComponent(this.value)" 
-      />
```

Comment:
This looks like form validation is removed. Deliberate?

Reply:
See above